### PR TITLE
Fix some ESLint errors

### PR DESCRIPTION
**Pull request by costinsin**
## Type of change

-   [x] 🧑‍💻 Improvement

## Description

Fixes some of the errors reported by ESLint.

## Checklist

-   [x] My code follows the contributor guidelines of this project;
-   [x] New and existing unit tests pass locally with my changes;

### CODE REVIEW

**Comment by costinsin** [comment]
Current dependencies on/for this PR:
* dev
  * **PR #546** <a href="https://app.graphite.dev/github/pr/Genez-io/genezio/546?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/Genez-io/genezio/546?utm_source=stack-comment).

File: src/generateSdk/sdkGeneratorHandler.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export async function generateSdk(
 
     if (plugins) {
         pluginsImported = plugins?.map(async (plugin) => {
-            return await import(plugin).catch((err: any) => {
+            return await import(plugin).catch(() => {
```
Comment (andreia-oca) [inline src/generateSdk/sdkGeneratorHandler.ts:28]: Here do not remove `err`, display the error for the verbose output.

You can do that with `debugLogger.debug()`.

File: src/telemetry/telemetry.ts [hunk 2]
```diff
@@ -72,7 +72,7 @@ export class GenezioTelemetry {
         const userLanguage: string = Intl.DateTimeFormat().resolvedOptions().locale;
         // get user operating system
         const operatingSystem: string = process.platform;
-        const sessionId: string = await this.getSessionId().catch((err) => {
+        const sessionId: string = await this.getSessionId().catch(() => {
```
Comment (andreia-oca) [inline src/telemetry/telemetry.ts:75]: Here the same comment on keeping `err` and displayinh it in the verbose output.

**Review by andreia-oca — APPROVED** [review]
Looks good! After adding the error debug logs, you can safely merge this.

**Pull request merged**